CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Change Log
 
+## [v1.1.7](https://github.com/ably/ably-ruby/tree/v1.1.7)
+
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.1.6...HEAD)
```

Comment:
This line needs to be ready for the new tag after you land this to `main` branch.

```suggestion
[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.1.6...v1.1.7)
```